src/main/java/org/mifos/connector/phee/zeebe/ZeebeVariables.java

```diff
@@ -115,6 +115,6 @@ private ZeebeVariables(){}
     public static final String HEADER_BILLER_ID = "X-Biller-Id";
     public static final String HEADER_CLIENTCORRELATIONID = "X-Client-Correlation-ID";
     public static final String DEBULKINGDFSPID = "debulkingDfspid";
-
+    public static final String PAYEE_DFSP_ID = "X-PayeeDFSP-ID";
```

Comment:
use this naming convention only for API header. don't use it for Zeebe variable

Reply:
changed it to payeeDFSPId